native/src/components/PoiDetails.tsx

```diff
@@ -89,32 +81,29 @@ const PoiDetails: React.FC<PoiDetailsProps> = ({
 
   return (
     <PoiDetailsContainer>
+      <StyledText>{title}</StyledText>
+      {distance && <StyledDistance>{t('distanceKilometre', { distance })}</StyledDistance>}
```

Comment:
Distance shows wrong values for me, not sure if related to this PR. Set emulator location to augsburg, but distances are over 9000km.

Reply:
Perhaps we mixed longitude and latitude again

Reply:
You sure? For me it works fine: 
<img src="https://user-images.githubusercontent.com/29095487/165119229-67caef5c-3218-4508-b29f-9057244db11f.png" width="400px" />